cli/unstable_prompt_select.ts

```diff
@@ -44,8 +55,13 @@ export function promptSelect(
 ): string | null {
   if (!input.isTerminal()) return null;
 
+  const { visibleLines = 3, indicator = "❯" } = options;
```

Comment:
3 as default feels too small to me as most terminal has much more lines. How about making default to `values.length` (keeping the existing behavior)?